pkg/backup/item_backupper.go

```diff
@@ -320,7 +320,9 @@ func (ib *itemBackupper) executeActions(
 		}
 		log.Info("Executing custom action")
 
-		updatedItem, additionalItemIdentifiers, err := action.Execute(obj, ib.backupRequest.Backup)
+		// Note: we're ignoring the operationID returned from Execute for now, it will be used
+		// with the async plugin action implementation
+		updatedItem, additionalItemIdentifiers, _, err := action.Execute(obj, ib.backupRequest.Backup)
```

Comment:
After the `Execute` method is called, the method may return in a short time but the async operation is still ongoing. 
As to the backup workflow, I guess the caller need to call the `Progress` method to wait the completion of the async operation.
For different items, multiple `Execute` methods may be called successively before their async operations complete, so before the backup workflow ends, multiple async operations should be waited for completion.

Will we handle this logic in the current PR or we do it in later PRs?

Reply:
@Lyndon-Li This PR is just the API definition. It adds the appropriate functions to the various BackupItemAction-related packages. Implmeenting the controller workflow changes to actually call Progress, wait, maintain a list of items waited for, etc. will be implemented in a separate PR (there's a separate github issue to track that work).